lab_1/main.py

```diff
@@ -49,7 +68,15 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    top_n_words = []
+    if isinstance(freq_dict, dict) and isinstance(top_n, int):
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.